tests/components/progettihwsw/test_config_flow.py

```diff
@@ -111,58 +87,49 @@ async def test_form_cannot_connect(hass):
     assert result2["errors"] == {"base": "cannot_connect"}
 
 
-async def test_form_user_exception(hass):
-    """Test we handle unknown exception."""
+async def test_form_existing_entry_exception(hass):
+    """Test we handle existing board."""
     result = await hass.config_entries.flow.async_init(
         DOMAIN, context={"source": config_entries.SOURCE_USER}
     )
 
     assert result["step_id"] == "user"
 
-    with patch(
-        "homeassistant.components.progettihwsw.config_flow.validate_input",
-        side_effect=Exception,
-    ):
-        result2 = await hass.config_entries.flow.async_configure(
-            result["flow_id"],
-            {CONF_HOST: "", CONF_PORT: 80},
-        )
+    entry = MockConfigEntry(
+        domain=DOMAIN,
+        data={
+            CONF_HOST: "",
+            CONF_PORT: 80,
+        },
+    )
+    entry.add_to_hass(hass)
 
-    assert result2["type"] == RESULT_TYPE_FORM
-    assert result2["step_id"] == "user"
-    assert result2["errors"] == {"base": "unknown"}
+    result2 = await hass.config_entries.flow.async_configure(
+        result["flow_id"],
+        {CONF_HOST: "", CONF_PORT: 80},
+    )
+
+    assert result2["type"] == RESULT_TYPE_ABORT
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6103615)